java/arcs/core/storage/driver/Volatile.kt

```diff
@@ -44,7 +50,15 @@ data class VolatileDriverProvider(private val arcId: ArcId) : DriverProvider {
         return VolatileDriver(storageKey, arcMemory)
     }
 
-    override suspend fun getAllStorageKeys(): Set<StorageKey> = arcMemory.keys()
+    override suspend fun getAllStorageKeys(): Map<StorageKey, Type> {
+        // TODO: keep track of and return the actual schema type.
+        val type = EntityType(Schema(
+            listOf<SchemaName>(),
+            SchemaFields(emptyMap(), emptyMap()),
+            ""
+        ))
+        return arcMemory.keys().map { it to CollectionType(type) }.toMap()
```

Comment:
Is every key guaranteed to be a collection type?

Reply:
no, but i added a todo and prefer to do it in a follow up pr, if that's ok.